index.html

```diff
@@ -22,24 +22,160 @@
     <h1>I'm Majek Glad to see you!</h1>
     <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
     <h3 class="links">LET'S CONNECT</h3>
-  
+  </section>
     <!--Social media accounts-->
+  <footer>
     <ul class="social-icons">
       <li><a href="#"><img src="images/Twitter.jpg" alt="twitter"></a></li>
       <li><a href="#"><img src="images/Linkedin icon.jpg" alt="Linkedin"></a></li>
       <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
       <li><a href="#"><img src="images/Githhub.jpg" alt="Github"></a></li>
     </ul>
-  </section>
-</body>
-</html>
-
+  </footer>
 
+  <!--works section-->
 
+  <div class="page-2">
+    <div class="container">
+      <a href="#"><img src="images/Snapshoot Portfolio.jpg"></a>
+      <div id="text">
+        <a href="#"><img src="images/Project Title.jpg"></a>  
```

Comment:
For better semantics, I think it would be more appropriate to use a heading (`hx`) tag for the titles of the projects rather than a link tag or an image.